admin/javascript/src/SilverStripeComponent.js

```diff
@@ -1,55 +0,0 @@
-/**
```

Comment:
Yeah, we followed the WP Calypso directory and file naming conventions here, which makes sense for "components with sub-components in the same folder". I slightly prefer the "after" variation below (complying with AirBnB), but the folder/file name duplication still irks me. Can we merge this without a major file rename please, and defer the discussion? The PR is already big enough ;)

```
// before
src/
  components/
    grid-field/
      index.js
      header.js
      styles.css
      README.md

// after
  components/
    GridField/
      GridField.js
      GridFieldHeader.js
      styles.css
      README.md
```